apps/11gist-blog/script.js

```diff
@@ -1,5 +1,35 @@
+
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+    // You code here
+    $.ajax('https://api.github.com/users/jroman93/gists', {
+        success: function(posts) {
+            posts.forEach(function(post) {
+                if (post.description.startsWith('#post')) {
+                    var strPostDescription = post.description.slice(5);
+                    var $post = $('<li><a href="#" data-url="' + post.url + '" data-comments="' + post.comments_url + '">' + strPostDescription + '</a></li>');
+                    $('#posts').append($post);
+                };
+            })
+            $('body').on('click', 'a', function(event) {
```

Comment:
Interesting choice here. Can you think of another way to write this $().on call? This is slightly different from the approaches we've used in class